### PR TITLE
🌐 Fix emojis and illustrations in Portuguese translation for `docs/pt/docs/async.md`

### PR DESCRIPTION
**Pull request by flaviofilipe**
Fixing PT documentation page "Concurrency and async / await"
- Showing emojis
- Showing images

### CODE REVIEW

File: docs/pt/docs/async.md [hunk 1]
```diff
@@ -102,107 +102,140 @@ Para ver essa diferença, imagine a seguinte história sobre hambúrgueres:
...
 
 Você fica na fila enquanto alguns (vamos dizer 8) caixas pegam os pedidos das pessoas na sua frente.
 
 Todo mundo antes de você está esperando pelos hambúrgueres estarem prontos antes de deixar o caixa porquê cada um dos 8 caixas vai e prepara o hambúrguer antes de pegar o próximo pedido.
 
-Então é finalmente sua vez, e pede 2 hambúrgueres muito saborosos para você e seu _crush_ :heart_eyes:.
+<img src="/img/async/parallel-burgers/parallel-burgers-01.png" class="illustration">
 
-Você paga.
+Então é finalmente sua vez, e pede 2 hambúrgueres muito saborosos para você e seu _crush_ 😍.
```
Comment (LorhanSohaky) [inline docs/pt/docs/async.md:170]:
```suggestion
Então finalmente é sua vez, e pede 2 hambúrgueres muito saborosos para você e seu _crush_ 😍.
```

**Review by LorhanSohaky — COMMENTED** [review]
Creio que com essas sugestões a leitura fique mais fluida

**Review by peidrao — COMMENTED** [review]
Como todo respeito, mas qual o sentidos de tantos emojis?

**Comment by LorhanSohaky** [comment]
> Como todo respeito, mas qual o sentidos de tantos emojis?

Isso vem de como a documentação original foi escrita. Uma escolha do autor

**Comment by tiangolo** [comment]
📝 Docs preview for commit a2eb0548e9b10c00767af802b77383778bfe7c0d at: https://649a1e947e9b76169fba4df1--fastapi.netlify.app

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**